pyproject.toml

```diff
@@ -9,13 +9,14 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.8"
-whylogs = {version = "^1.1.40"}
+whylogs = {version = "1.1.42.dev0"}
 textstat = "^0.7.3"
 sentence-transformers = "^2.2.2"
 nltk = "^3.8.1"
 openai = "^0.27.6"
 pandas = "*"
 datasets = "^2.12.0"
+torch = "*"
```

Comment:
Can we make torch an optional dependency? Did you need it for testing?

Reply:
We can probably make it optional. The UDF is using torch to compute the cosine similarity, so I had to pull it in to get the tests to pass.